doozer/doozerlib/backend/rebaser.py

```diff
@@ -954,12 +969,40 @@ def _add_build_repos(self, dfp: DockerfileParser, metadata: ImageMetadata, dest_
             all_stages=True,
         )
 
+        last_stage = self.split_dockerfile_into_stages(dfp)[-1]
+
+        # Find all the USERs in the last stage
+        user_stanzas = []
+        for line in last_stage:
+            if "USER" in line.keys():
+                user_stanzas.append(f"USER {line['USER']}")
```

Comment:
Can this be simplified to
```suggestion
                final_stage_user = f"USER {line['USER']}"
```

Then later:
```
if final_stage_user.split()[-1] == '0':
    final_stage_user = None  # Avoid redundant USER 0 statement after repo removal
```

Reply:
updated in https://github.com/openshift-eng/art-tools/pull/1452/commits/f40809e03f9cad43ab1b6bf40583d729189280ee